utils/middlewares/userDataProcessing.ts

```diff
@@ -0,0 +1,48 @@
+import { ResponseErrorFromValidationErrors } from "italia-ts-commons/lib/responses";
+
+import { UserDataProcessingChoiceRequest } from "io-functions-commons/dist/generated/definitions/UserDataProcessingChoiceRequest";
+import {
+  NewUserDataProcessing,
+  UserDataProcessing
+} from "io-functions-commons/dist/src/models/user_data_processing";
+import { IRequestMiddleware } from "io-functions-commons/dist/src/utils/request_middleware";
+
+/**
+ * A middleware that extracts a NewUserDataProcessing payload from a request.
+ */
+export const NewUserDataProcessingMiddleware: IRequestMiddleware<
```

Comment:
I've added a middleware that you can use inline to parse body payload:
https://github.com/pagopa/io-functions-commons/blob/master/src/utils/middlewares/required_body_payload.ts

can you try using this (1.12.0 version of io-functions-commons package) and get rid of this new middlewares here ?

Reply:
Do you mean refactoring this way?
```
import { UserDataProcessingChoiceRequest } from "io-functions-commons/dist/generated/definitions/UserDataProcessingChoiceRequest";
import {
  NewUserDataProcessing,
  UserDataProcessing
} from "io-functions-commons/dist/src/models/user_data_processing";
import { RequiredBodyPayloadMiddleware } from "io-functions-commons/dist/src/utils/middlewares/required_body_payload";
/**
 * A middleware that extracts a NewUserDataProcessing payload from a request.
 */
export const NewUserDataProcessingMiddleware = RequiredBodyPayloadMiddleware(
  NewUserDataProcessing
);

/**
 * A middleware that extracts a UserDataProcessingChoice payload from a request.
 */
export const UserDataProcessingChoiceMiddleware = RequiredBodyPayloadMiddleware(
  UserDataProcessingChoiceRequest
);

/**
 * A middleware that extracts a UserDataProcessing payload from a request.
 */
export const UserDataProcessingMiddleware = RequiredBodyPayloadMiddleware(
  UserDataProcessing
);
```

Reply:
no I mean use it inline in the handler.ts without assign new names
(the usage is right anyway : )